src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
```

Comment:
this could be written easier

```suggestion
export const Person = ({
  person: {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  },
}) => {
```

Reply:
Anna's variant is mush better, but your's.
@andrii-dmytruk what do you want to tell by this code? Are you writing the type of Object person? if it is so use interface or better generic for such purpose)